### ISSUE TITLE
[BUG] Can't install module

### ISSUE THREAD

**Issue by aurelien-m** [daquexian/onnx-simplifier#205]
I'm not able to install `onnxsim`.

Here's what I've tried:

* `pip install onnx-simplifier`

```
Looking in indexes: https://pypi.org/simple, https://pypi.ngc.nvidia.com
Collecting onnx-simplifier
  Downloading onnx-simplifier-0.4.1.tar.gz (19.8 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 19.8/19.8 MB 22.8 MB/s eta 0:00:00
  Preparing metadata (setup.py) ... done
  WARNING: Generating metadata for package onnx-simplifier produced metadata for project name onnxsim. Fix your #egg=onnx-simplifier fragments.
Discarding https://files.pythonhosted.org/packages/7e/b7/e466aa2b2b74333e4d430a862ec53a83047b07315d9aaa7f07aba1fa7e09/onnx-simplifier-0.4.1.tar.gz#sha256=2b7af644d70bf46d60ad995333ccf7b1df24bc548408d6b9efee6c790f419643 (from https://pypi.org/simple/onnx-simplifier/) (requires-python:>=3.6): Requested onnxsim from https://files.pythonhosted.org/packages/7e/b7/e466aa2b2b74333e4d430a862ec53a83047b07315d9aaa7f07aba1fa7e09/onnx-simplifier-0.4.1.tar.gz#sha256=2b7af644d70bf46d60ad995333ccf7b1df24bc548408d6b9efee6c790f419643 has inconsistent name: filename has 'onnx-simplifier', but metadata has 'onnxsim'
  Downloading onnx-simplifier-0.4.0.tar.gz (6.8 kB)
  Preparing metadata (setup.py) ... done
Collecting onnxsim-no-ort==0.4.0
  Downloading onnxsim-no-ort-0.4.0.tar.gz (28.6 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 28.6/28.6 MB 25.2 MB/s eta 0:00:00
  Preparing metadata (setup.py) ... done
  WARNING: Requested onnxsim-no-ort==0.4.0 from https://files.pythonhosted.org/packages/a5/03/9f571821ae47b5192b415a4010738877fe69adb6271ea4c182abd3fe51a8/onnxsim-no-ort-0.4.0.tar.gz#sha256=0ffb19bf4a1f894540f06f1a85c9d97b55ce02e9924150239c5429b7326e2e90 (from onnx-simplifier), but installing version unknown
Discarding https://files.pythonhosted.org/packages/a5/03/9f571821ae47b5192b415a4010738877fe69adb6271ea4c182abd3fe51a8/onnxsim-no-ort-0.4.0.tar.gz#sha256=0ffb19bf4a1f894540f06f1a85c9d97b55ce02e9924150239c5429b7326e2e90 (from https://pypi.org/simple/onnxsim-no-ort/) (requires-python:>=3.6): Requested onnxsim-no-ort==0.4.0 from https://files.pythonhosted.org/packages/a5/03/9f571821ae47b5192b415a4010738877fe69adb6271ea4c182abd3fe51a8/onnxsim-no-ort-0.4.0.tar.gz#sha256=0ffb19bf4a1f894540f06f1a85c9d97b55ce02e9924150239c5429b7326e2e90 (from onnx-simplifier) has inconsistent version: filename has '0.4.0', but metadata has 'unknown'
Collecting onnx-simplifier
  Downloading onnx-simplifier-0.3.10.tar.gz (14 kB)
  Preparing metadata (setup.py) ... done
Requirement already satisfied: onnx in /home/aurelien/Projects/Delair/AI/aienv/lib/python3.10/site-packages (from onnx-simplifier) (1.11.0)
Collecting onnxoptimizer>=0.2.6
  Downloading onnxoptimizer-0.2.7.tar.gz (16.8 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 16.8/16.8 MB 21.0 MB/s eta 0:00:00
  Preparing metadata (setup.py) ... done
Requirement already satisfied: protobuf>=3.7.0 in /home/aurelien/Projects/Delair/AI/aienv/lib/python3.10/site-packages (from onnx-simplifier) (3.20.1)
Requirement already satisfied: rich in /home/aurelien/Projects/Delair/AI/aienv/lib/python3.10/site-packages (from onnx-simplifier) (12.4.4)
Collecting onnx-simplifier
  Downloading onnx-simplifier-0.3.9.tar.gz (14 kB)
  Preparing metadata (setup.py) ... done
  Downloading onnx-simplifier-0.3.8.tar.gz (14 kB)
  Preparing metadata (setup.py) ... done
  Downloading onnx-simplifier-0.3.7.tar.gz (13 kB)
  Preparing metadata (setup.py) ... done
  Downloading onnx-simplifier-0.3.6.tar.gz (13 kB)
  Preparing metadata (setup.py) ... done
  Downloading onnx-simplifier-0.3.5.tar.gz (13 kB)
  Preparing metadata (setup.py) ... done
  Downloading onnx-simplifier-0.3.4.tar.gz (9.0 kB)
  Preparing metadata (setup.py) ... done
  Downloading onnx-simplifier-0.3.3.tar.gz (9.0 kB)
  Preparing metadata (setup.py) ... done
  Downloading onnx-simplifier-0.3.2.tar.gz (8.9 kB)
  Preparing metadata (setup.py) ... done
  Downloading onnx-simplifier-0.3.1.tar.gz (8.8 kB)
  Preparing metadata (setup.py) ... done
  Downloading onnx-simplifier-0.3.0.tar.gz (8.8 kB)
  Preparing metadata (setup.py) ... done
  Downloading onnx-simplifier-0.2.28.tar.gz (8.4 kB)
  Preparing metadata (setup.py) ... done
  Downloading onnx-simplifier-0.2.27.tar.gz (8.2 kB)
  Preparing metadata (setup.py) ... done
  Downloading onnx-simplifier-0.2.26.tar.gz (8.3 kB)
  Preparing metadata (setup.py) ... done
  Downloading onnx-simplifier-0.2.25.tar.gz (8.2 kB)
  Preparing metadata (setup.py) ... done
  Downloading onnx-simplifier-0.2.24.tar.gz (8.0 kB)
  Preparing metadata (setup.py) ... done
  Downloading onnx-simplifier-0.2.23.tar.gz (6.9 kB)
  Preparing metadata (setup.py) ... done
  Downloading onnx-simplifier-0.2.22.tar.gz (7.1 kB)
  Preparing metadata (setup.py) ... done
  Downloading onnx-simplifier-0.2.21.tar.gz (7.0 kB)
  Preparing metadata (setup.py) ... done
  Downloading onnx-simplifier-0.2.20.tar.gz (7.0 kB)
  Preparing metadata (setup.py) ... done
  Downloading onnx-simplifier-0.2.19.tar.gz (7.3 kB)
  Preparing metadata (setup.py) ... done
  Downloading onnx-simplifier-0.2.18.tar.gz (7.3 kB)
  Preparing metadata (setup.py) ... done
  Downloading onnx-simplifier-0.2.17.tar.gz (7.3 kB)
  Preparing metadata (setup.py) ... done
  Downloading onnx-simplifier-0.2.16.tar.gz (7.2 kB)
  Preparing metadata (setup.py) ... done
  Downloading onnx-simplifier-0.2.15.tar.gz (7.2 kB)
  Preparing metadata (setup.py) ... done
  Downloading onnx-simplifier-0.2.14.tar.gz (7.2 kB)
  Preparing metadata (setup.py) ... done
  Downloading onnx-simplifier-0.2.13.tar.gz (7.2 kB)
  Preparing metadata (setup.py) ... done
  Downloading onnx-simplifier-0.2.12.tar.gz (7.1 kB)
  Preparing metadata (setup.py) ... done
  Downloading onnx-simplifier-0.2.11.tar.gz (7.0 kB)
  Preparing metadata (setup.py) ... done
  Downloading onnx-simplifier-0.2.10.tar.gz (6.9 kB)
  Preparing metadata (setup.py) ... done
  Downloading onnx-simplifier-0.2.9.tar.gz (7.0 kB)
  Preparing metadata (setup.py) ... done
  Downloading onnx-simplifier-0.2.8.tar.gz (6.6 kB)
  Preparing metadata (setup.py) ... done
  Downloading onnx-simplifier-0.2.7.tar.gz (6.6 kB)
  Preparing metadata (setup.py) ... done
  Downloading onnx-simplifier-0.2.6.tar.gz (6.5 kB)
  Preparing metadata (setup.py) ... done
  Downloading onnx-simplifier-0.2.5.tar.gz (6.4 kB)
  Preparing metadata (setup.py) ... done
  Downloading onnx-simplifier-0.2.4.tar.gz (6.3 kB)
  Preparing metadata (setup.py) ... done
  Downloading onnx-simplifier-0.2.3.tar.gz (6.0 kB)
  Preparing metadata (setup.py) ... done
  Downloading onnx-simplifier-0.2.2.tar.gz (5.8 kB)
  Preparing metadata (setup.py) ... done
  Downloading onnx-simplifier-0.2.1.tar.gz (5.8 kB)
  Preparing metadata (setup.py) ... done
  Downloading onnx-simplifier-0.2.0.tar.gz (5.7 kB)
  Preparing metadata (setup.py) ... done
  Downloading onnx-simplifier-0.1.9.tar.gz (5.5 kB)
  Preparing metadata (setup.py) ... done
  Downloading onnx-simplifier-0.1.8.tar.gz (4.7 kB)
  Preparing metadata (setup.py) ... done
  Downloading onnx-simplifier-0.1.6.tar.gz (4.6 kB)
  Preparing metadata (setup.py) ... done
  Downloading onnx-simplifier-0.1.5.tar.gz (4.5 kB)
  Preparing metadata (setup.py) ... done
  Downloading onnx-simplifier-0.1.4.tar.gz (4.4 kB)
  Preparing metadata (setup.py) ... done
  Downloading onnx-simplifier-0.1.3.tar.gz (4.3 kB)
  Preparing metadata (setup.py) ... done
  Downloading onnx-simplifier-0.1.2.tar.gz (4.2 kB)
  Preparing metadata (setup.py) ... done
  Downloading onnx_simplifier-0.1.1-py3-none-any.whl (8.1 kB)
  Downloading onnx_simplifier-0.1.0-py3-none-any.whl (8.1 kB)
  Downloading onnx-simplifier-0.0.0.tar.gz (4.6 kB)
  Preparing metadata (setup.py) ... done
ERROR: Cannot install onnx-simplifier==0.0.0, onnx-simplifier==0.1.0, onnx-simplifier==0.1.1, onnx-simplifier==0.1.2, onnx-simplifier==0.1.3, onnx-simplifier==0.1.4, onnx-simplifier==0.1.5, onnx-simplifier==0.1.6, onnx-simplifier==0.1.8, onnx-simplifier==0.1.9, onnx-simplifier==0.2.0, onnx-simplifier==0.2.1, onnx-simplifier==0.2.10, onnx-simplifier==0.2.11, onnx-simplifier==0.2.12, onnx-simplifier==0.2.13, onnx-simplifier==0.2.14, onnx-simplifier==0.2.15, onnx-simplifier==0.2.16, onnx-simplifier==0.2.17, onnx-simplifier==0.2.18, onnx-simplifier==0.2.19, onnx-simplifier==0.2.2, onnx-simplifier==0.2.20, onnx-simplifier==0.2.21, onnx-simplifier==0.2.22, onnx-simplifier==0.2.23, onnx-simplifier==0.2.24, onnx-simplifier==0.2.25, onnx-simplifier==0.2.26, onnx-simplifier==0.2.27, onnx-simplifier==0.2.28, onnx-simplifier==0.2.3, onnx-simplifier==0.2.4, onnx-simplifier==0.2.5, onnx-simplifier==0.2.6, onnx-simplifier==0.2.7, onnx-simplifier==0.2.8, onnx-simplifier==0.2.9, onnx-simplifier==0.3.0, onnx-simplifier==0.3.1, onnx-simplifier==0.3.10, onnx-simplifier==0.3.2, onnx-simplifier==0.3.3, onnx-simplifier==0.3.4, onnx-simplifier==0.3.5, onnx-simplifier==0.3.6, onnx-simplifier==0.3.7, onnx-simplifier==0.3.8, onnx-simplifier==0.3.9 and onnx-simplifier==0.4.0 because these package versions have conflicting dependencies.

The conflict is caused by:
    onnx-simplifier 0.4.0 depends on onnxsim-no-ort==0.4.0
    onnx-simplifier 0.3.10 depends on onnxruntime>=1.11.1
    onnx-simplifier 0.3.9 depends on onnxruntime>=1.10.0
    onnx-simplifier 0.3.8 depends on onnxruntime>=1.10.0
    onnx-simplifier 0.3.7 depends on onnxruntime>=1.6.0
    onnx-simplifier 0.3.6 depends on onnxruntime>=1.6.0
    onnx-simplifier 0.3.5 depends on onnxruntime>=1.6.0
    onnx-simplifier 0.3.4 depends on onnxruntime>=1.6.0
    onnx-simplifier 0.3.3 depends on onnxruntime>=1.6.0
    onnx-simplifier 0.3.2 depends on onnxruntime>=1.6.0
    onnx-simplifier 0.3.1 depends on onnxruntime>=1.6.0
    onnx-simplifier 0.3.0 depends on onnxruntime>=1.6.0
    onnx-simplifier 0.2.28 depends on onnxruntime>=1.6.0
    onnx-simplifier 0.2.27 depends on onnxruntime>=1.6.0
    onnx-simplifier 0.2.26 depends on onnxruntime-noopenmp>=1.6.0
    onnx-simplifier 0.2.25 depends on onnxruntime>=1.6.0
    onnx-simplifier 0.2.24 depends on onnxruntime>=1.6.0
    onnx-simplifier 0.2.23 depends on onnxruntime>=1.6.0
    onnx-simplifier 0.2.22 depends on onnxruntime>=1.6.0
    onnx-simplifier 0.2.21 depends on onnxruntime>=1.6.0
    onnx-simplifier 0.2.20 depends on onnxruntime>=1.6.0
    onnx-simplifier 0.2.19 depends on onnxruntime>=1.2.0
    onnx-simplifier 0.2.18 depends on onnxruntime>=1.2.0
    onnx-simplifier 0.2.17 depends on onnxruntime>=1.2.0
    onnx-simplifier 0.2.16 depends on onnxruntime>=1.2.0
    onnx-simplifier 0.2.15 depends on onnxruntime>=1.2.0
    onnx-simplifier 0.2.14 depends on onnxruntime>=1.2.0
    onnx-simplifier 0.2.13 depends on onnxruntime>=1.2.0
    onnx-simplifier 0.2.12 depends on onnxruntime>=1.2.0
    onnx-simplifier 0.2.11 depends on onnxruntime>=1.2.0
    onnx-simplifier 0.2.10 depends on onnxruntime>=1.2.0
    onnx-simplifier 0.2.9 depends on onnxruntime>=1.2.0
    onnx-simplifier 0.2.8 depends on onnxruntime>=1.2.0
    onnx-simplifier 0.2.7 depends on onnxruntime>=1.2.0
    onnx-simplifier 0.2.6 depends on onnxruntime>=0.3.0
    onnx-simplifier 0.2.5 depends on onnxruntime>=0.3.0
    onnx-simplifier 0.2.4 depends on onnxruntime>=0.3.0
    onnx-simplifier 0.2.3 depends on onnxruntime>=0.3.0
    onnx-simplifier 0.2.2 depends on onnxruntime>=0.3.0
    onnx-simplifier 0.2.1 depends on onnxruntime>=0.3.0
    onnx-simplifier 0.2.0 depends on onnxruntime>=0.3.0
    onnx-simplifier 0.1.9 depends on onnxruntime>=0.3.0
    onnx-simplifier 0.1.8 depends on onnxruntime>=0.3.0
    onnx-simplifier 0.1.6 depends on onnxruntime>=0.3.0
    onnx-simplifier 0.1.5 depends on onnxruntime>=0.3.0
    onnx-simplifier 0.1.4 depends on onnxruntime>=0.3.0
    onnx-simplifier 0.1.3 depends on onnxruntime>=0.3.0
    onnx-simplifier 0.1.2 depends on onnxruntime>=0.3.0
    onnx-simplifier 0.1.1 depends on onnxruntime>=0.3.0
    onnx-simplifier 0.1.0 depends on onnxruntime>=0.3.0
    onnx-simplifier 0.0.0 depends on onnxruntime>=0.3.0

To fix this you could try to:
1. loosen the range of package versions you've specified
2. remove package versions to allow pip attempt to solve the dependency conflict

ERROR: ResolutionImpossible: for help visit https://pip.pypa.io/en/latest/topics/dependency-resolution/#dealing-with-dependency-conflicts
WARNING: There was an error checking the latest version of pip.
```

* `pip install onnxsim`

```
Looking in indexes: https://pypi.org/simple, https://pypi.ngc.nvidia.com
Collecting onnxsim
  Downloading onnxsim-0.4.1.tar.gz (19.8 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 19.8/19.8 MB 21.8 MB/s eta 0:00:00
  Preparing metadata (setup.py) ... done
Discarding https://files.pythonhosted.org/packages/23/f0/8bd92e59a5c2d29f5e67bfe41e942f5edd592b98041aa819963e1c8ccde5/onnxsim-0.4.1.tar.gz#sha256=3a8c178ea48ee313550bc111fa2d2cfdcb1178abb596c0719461ed840118cc5b (from https://pypi.org/simple/onnxsim/) (requires-python:>=3.6): Requested onnxsim from https://files.pythonhosted.org/packages/23/f0/8bd92e59a5c2d29f5e67bfe41e942f5edd592b98041aa819963e1c8ccde5/onnxsim-0.4.1.tar.gz#sha256=3a8c178ea48ee313550bc111fa2d2cfdcb1178abb596c0719461ed840118cc5b has inconsistent version: filename has '0.4.1', but metadata has 'unknown'
  Downloading onnxsim-0.4.0.tar.gz (6.8 kB)
  Preparing metadata (setup.py) ... done
Collecting onnxsim-no-ort==0.4.0
  Downloading onnxsim-no-ort-0.4.0.tar.gz (28.6 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 28.6/28.6 MB 24.9 MB/s eta 0:00:00
  Preparing metadata (setup.py) ... done
  WARNING: Requested onnxsim-no-ort==0.4.0 from https://files.pythonhosted.org/packages/a5/03/9f571821ae47b5192b415a4010738877fe69adb6271ea4c182abd3fe51a8/onnxsim-no-ort-0.4.0.tar.gz#sha256=0ffb19bf4a1f894540f06f1a85c9d97b55ce02e9924150239c5429b7326e2e90 (from onnxsim), but installing version unknown
Discarding https://files.pythonhosted.org/packages/a5/03/9f571821ae47b5192b415a4010738877fe69adb6271ea4c182abd3fe51a8/onnxsim-no-ort-0.4.0.tar.gz#sha256=0ffb19bf4a1f894540f06f1a85c9d97b55ce02e9924150239c5429b7326e2e90 (from https://pypi.org/simple/onnxsim-no-ort/) (requires-python:>=3.6): Requested onnxsim-no-ort==0.4.0 from https://files.pythonhosted.org/packages/a5/03/9f571821ae47b5192b415a4010738877fe69adb6271ea4c182abd3fe51a8/onnxsim-no-ort-0.4.0.tar.gz#sha256=0ffb19bf4a1f894540f06f1a85c9d97b55ce02e9924150239c5429b7326e2e90 (from onnxsim) has inconsistent version: filename has '0.4.0', but metadata has 'unknown'
ERROR: Could not find a version that satisfies the requirement onnxsim-no-ort==0.4.0 (from onnxsim) (from versions: 0.4.0)
ERROR: No matching distribution found for onnxsim-no-ort==0.4.0
```

* Install from source: `python setup.py install`

```
/home/aurelien/Projects/Delair/AI/aienv/lib/python3.10/site-packages/setuptools/installer.py:27: SetuptoolsDeprecationWarning: setuptools.installer is deprecated. Requirements should be satisfied by a PEP 517 installer.
  warnings.warn(
running install
/home/aurelien/Projects/Delair/AI/aienv/lib/python3.10/site-packages/setuptools/command/install.py:34: SetuptoolsDeprecationWarning: setup.py install is deprecated. Use build and pip and other standards-based tools.
  warnings.warn(
/home/aurelien/Projects/Delair/AI/aienv/lib/python3.10/site-packages/setuptools/command/easy_install.py:144: EasyInstallDeprecationWarning: easy_install command is deprecated. Use build and pip and other standards-based tools.
  warnings.warn(
running bdist_egg
running egg_info
writing onnxsim.egg-info/PKG-INFO
writing dependency_links to onnxsim.egg-info/dependency_links.txt
writing entry points to onnxsim.egg-info/entry_points.txt
writing requirements to onnxsim.egg-info/requires.txt
writing top-level names to onnxsim.egg-info/top_level.txt
reading manifest file 'onnxsim.egg-info/SOURCES.txt'
reading manifest template 'MANIFEST.in'
warning: no files found matching '*.c' under directory 'onnxsim'
warning: no files found matching '*.proto' under directory 'onnxsim'
warning: no files found matching '*' under directory 'third_party'
warning: no previously-included files matching '*' found under directory 'third_party/onnxruntime'
warning: no previously-included files matching '*' found under directory 'third_party/onnx-optimizer/build'
warning: no previously-included files matching '*' found under directory 'third_party/onnx/build'
warning: no previously-included files matching '*' found under directory 'third_party/onnx/onnx/backend'
adding license file 'LICENSE'
writing manifest file 'onnxsim.egg-info/SOURCES.txt'
installing library code to build/bdist.linux-x86_64/egg
running install_lib
running build_py
running create_version
copying onnxsim/version.py -> build/lib.linux-x86_64-cpython-310/onnxsim
/home/aurelien/Projects/Delair/AI/aienv/lib/python3.10/site-packages/setuptools/command/build_py.py:153: SetuptoolsDeprecationWarning:     Installing 'onnxsim.bin' as data is deprecated, please list it in `packages`.
    !!


    ############################
    # Package would be ignored #
    ############################
    Python recognizes 'onnxsim.bin' as an importable package, however it is
    included in the distribution as "data".
    This behavior is likely to change in future versions of setuptools (and
    therefore is considered deprecated).

    Please make sure that 'onnxsim.bin' is included as a package by using
    setuptools' `packages` configuration field or the proper discovery methods
    (for example by using `find_namespace_packages(...)`/`find_namespace:`
    instead of `find_packages(...)`/`find:`).

    You can read more about "package discovery" and "data files" on setuptools
    documentation page.


!!

  check.warn(importable)
running build_ext
running cmake_build
Run command ['/home/aurelien/Projects/Delair/AI/aienv/bin/cmake', '-DPython_INCLUDE_DIR=/usr/include/python3.10', '-DPython_EXECUTABLE=/home/aurelien/Projects/Delair/AI/aienv/bin/python', '-DPYTHON_EXECUTABLE=/home/aurelien/Projects/Delair/AI/aienv/bin/python', '-DBUILD_ONNX_PYTHON=OFF', '-DONNXSIM_PYTHON=ON', '-DONNXSIM_BUILTIN_ORT=OFF', '-DONNX_USE_LITE_PROTO=OFF', '-DCMAKE_EXPORT_COMPILE_COMMANDS=ON', '-DONNX_NAMESPACE=onnx', '-DPY_EXT_SUFFIX=.cpython-310-x86_64-linux-gnu.so', '-DONNX_OPT_USE_SYSTEM_PROTOBUF=OFF', '-DCMAKE_BUILD_TYPE=Release', '-DONNX_ML=1', '/home/aurelien/Projects/GitHub/onnx-simplifier']
CMake Error at CMakeLists.txt:42 (add_subdirectory):
  The source directory

    /home/aurelien/Projects/GitHub/onnx-simplifier/third_party/onnx-optimizer

  does not contain a CMakeLists.txt file.


CMake Error at CMakeLists.txt:70 (add_subdirectory):
  The source directory

    /home/aurelien/Projects/GitHub/onnx-simplifier/third_party/pybind11

  does not contain a CMakeLists.txt file.


CMake Error at CMakeLists.txt:71 (pybind11_add_module):
  Unknown CMake command "pybind11_add_module".


-- Configuring incomplete, errors occurred!
See also "/home/aurelien/Projects/GitHub/onnx-simplifier/.setuptools-cmake-build/CMakeFiles/CMakeOutput.log".
Traceback (most recent call last):
  File "/home/aurelien/Projects/GitHub/onnx-simplifier/setup.py", line 271, in <module>
    setuptools.setup(
  File "/home/aurelien/Projects/Delair/AI/aienv/lib/python3.10/site-packages/setuptools/__init__.py", line 87, in setup
    return distutils.core.setup(**attrs)
  File "/home/aurelien/Projects/Delair/AI/aienv/lib/python3.10/site-packages/setuptools/_distutils/core.py", line 148, in setup
    return run_commands(dist)
  File "/home/aurelien/Projects/Delair/AI/aienv/lib/python3.10/site-packages/setuptools/_distutils/core.py", line 163, in run_commands
    dist.run_commands()
  File "/home/aurelien/Projects/Delair/AI/aienv/lib/python3.10/site-packages/setuptools/_distutils/dist.py", line 967, in run_commands
    self.run_command(cmd)
  File "/home/aurelien/Projects/Delair/AI/aienv/lib/python3.10/site-packages/setuptools/dist.py", line 1229, in run_command
    super().run_command(command)
  File "/home/aurelien/Projects/Delair/AI/aienv/lib/python3.10/site-packages/setuptools/_distutils/dist.py", line 986, in run_command
    cmd_obj.run()
  File "/home/aurelien/Projects/Delair/AI/aienv/lib/python3.10/site-packages/setuptools/command/install.py", line 74, in run
    self.do_egg_install()
  File "/home/aurelien/Projects/Delair/AI/aienv/lib/python3.10/site-packages/setuptools/command/install.py", line 123, in do_egg_install
    self.run_command('bdist_egg')
  File "/home/aurelien/Projects/Delair/AI/aienv/lib/python3.10/site-packages/setuptools/_distutils/cmd.py", line 313, in run_command
    self.distribution.run_command(command)
  File "/home/aurelien/Projects/Delair/AI/aienv/lib/python3.10/site-packages/setuptools/dist.py", line 1229, in run_command
    super().run_command(command)
  File "/home/aurelien/Projects/Delair/AI/aienv/lib/python3.10/site-packages/setuptools/_distutils/dist.py", line 986, in run_command
    cmd_obj.run()
  File "/home/aurelien/Projects/Delair/AI/aienv/lib/python3.10/site-packages/setuptools/command/bdist_egg.py", line 165, in run
    cmd = self.call_command('install_lib', warn_dir=0)
  File "/home/aurelien/Projects/Delair/AI/aienv/lib/python3.10/site-packages/setuptools/command/bdist_egg.py", line 151, in call_command
    self.run_command(cmdname)
  File "/home/aurelien/Projects/Delair/AI/aienv/lib/python3.10/site-packages/setuptools/_distutils/cmd.py", line 313, in run_command
    self.distribution.run_command(command)
  File "/home/aurelien/Projects/Delair/AI/aienv/lib/python3.10/site-packages/setuptools/dist.py", line 1229, in run_command
    super().run_command(command)
  File "/home/aurelien/Projects/Delair/AI/aienv/lib/python3.10/site-packages/setuptools/_distutils/dist.py", line 986, in run_command
    cmd_obj.run()
  File "/home/aurelien/Projects/Delair/AI/aienv/lib/python3.10/site-packages/setuptools/command/install_lib.py", line 11, in run
    self.build()
  File "/home/aurelien/Projects/Delair/AI/aienv/lib/python3.10/site-packages/setuptools/_distutils/command/install_lib.py", line 107, in build
    self.run_command('build_ext')
  File "/home/aurelien/Projects/Delair/AI/aienv/lib/python3.10/site-packages/setuptools/_distutils/cmd.py", line 313, in run_command
    self.distribution.run_command(command)
  File "/home/aurelien/Projects/Delair/AI/aienv/lib/python3.10/site-packages/setuptools/dist.py", line 1229, in run_command
    super().run_command(command)
  File "/home/aurelien/Projects/Delair/AI/aienv/lib/python3.10/site-packages/setuptools/_distutils/dist.py", line 986, in run_command
    cmd_obj.run()
  File "/home/aurelien/Projects/GitHub/onnx-simplifier/setup.py", line 212, in run
    self.run_command('cmake_build')
  File "/home/aurelien/Projects/Delair/AI/aienv/lib/python3.10/site-packages/setuptools/_distutils/cmd.py", line 313, in run_command
    self.distribution.run_command(command)
  File "/home/aurelien/Projects/Delair/AI/aienv/lib/python3.10/site-packages/setuptools/dist.py", line 1229, in run_command
    super().run_command(command)
  File "/home/aurelien/Projects/Delair/AI/aienv/lib/python3.10/site-packages/setuptools/_distutils/dist.py", line 986, in run_command
    cmd_obj.run()
  File "/home/aurelien/Projects/GitHub/onnx-simplifier/setup.py", line 186, in run
    subprocess.check_call(cmake_args)
  File "/usr/lib/python3.10/subprocess.py", line 369, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['/home/aurelien/Projects/Delair/AI/aienv/bin/cmake', '-DPython_INCLUDE_DIR=/usr/include/python3.10', '-DPython_EXECUTABLE=/home/aurelien/Projects/Delair/AI/aienv/bin/python', '-DPYTHON_EXECUTABLE=/home/aurelien/Projects/Delair/AI/aienv/bin/python', '-DBUILD_ONNX_PYTHON=OFF', '-DONNXSIM_PYTHON=ON', '-DONNXSIM_BUILTIN_ORT=OFF', '-DONNX_USE_LITE_PROTO=OFF', '-DCMAKE_EXPORT_COMPILE_COMMANDS=ON', '-DONNX_NAMESPACE=onnx', '-DPY_EXT_SUFFIX=.cpython-310-x86_64-linux-gnu.so', '-DONNX_OPT_USE_SYSTEM_PROTOBUF=OFF', '-DCMAKE_BUILD_TYPE=Release', '-DONNX_ML=1', '/home/aurelien/Projects/GitHub/onnx-simplifier']' returned non-zero exit status 1.
```

Any ideas ? :) Thanks!

**Comment by daquexian** [comment]
Thanks for your report! What's your python version and os version?

**Comment by aurelien-m** [comment]
I'm using Python 3.10.4 on Pop OS 22.04

**Comment by daquexian** [comment]
I have published v0.4.2 with the support to python 3.10. Please try again, thanks!

**Comment by aurelien-m** [comment]
`pip install onnx-simplifier` worked perfectly. Thanks!
